### DISCUSSION TITLE
Progress bar or icon for reading / Associate with html files

### DISCUSSION THREAD

**Discussion by sirippon** [plateaukao/einkbro#207] [Ideas]
This is a wonderful browser on the Boox Note Air 2, thanks a lot! A couple of suggestions:

1) When reading long web pages, it would be great to have some indication on screen of how far through the text one currently is (Maybe a side scroll bar or a kindle style: n/nn showing the number of screens left still to read? Either could potentially be enhanced also to enable navigation to an arbitrary place on a long page, without any visible scrolling) 

2) einkbro doesn't seem to have the option to associate with .html files on the device and to open them. Could that be added? (It's a reasonably common use case, because people use Zotero to store .html snapshots of web pages and then want to open those to read on the device later.)

Thanks again for the wonderful app!

**Comment by plateaukao** [comment]
Thanks for the suggestions. I will take them into consideration.

**Comment by plateaukao** [comment]
EinkBro will support .mht and .mhtml format in next release.

**Reply by sirippon** [reply]
Awesome, thanks!

**Comment by plateaukao** [comment]
@sirippon 

page info feature will be available in toolbar settings in next version.

![image](https://user-images.githubusercontent.com/4084738/221248694-9207678e-a8c9-4d3a-91ab-18432a679cc7.png)

**Reply by amsitlab** [reply]
@plateaukao also toolbar for saved pages ( file.mht / file.mhtml) like bookmark toolbar with folder will be usefull
